<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsForm</class>
 <widget class="QDialog" name="settingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(20, 20, 20);</string>
  </property>
  <widget class="QLabel" name="SetLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>User information</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileImg_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="catImg">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.159204, y1:0.205, x2:1, y2:1, stop:0 rgba(100, 0, 221, 255), stop:1 rgba(255, 76, 255, 255));</string>
   </property>
   <property name="text">
    <string>ะะบ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string> Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string> Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Some information...</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileImg_div">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>profileImg_div</zorder>
  <zorder>SetLabel</zorder>
  <zorder>profileImg_2</zorder>
  <zorder>loginLabel</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>catImg</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
